canny:
  ros__parameters:
    thrs1: 600
    thrs2: 1500

canny_with_cam:
  ros__parameters:
    bright: 1
    contrast: 1.1
    blur: 5
    thrs1: 100
    thrs2: 200

